SubmissionTest.java

```diff
@@ -603,6 +603,272 @@ public void testGetCostumeCountForSpriteValid()
     }
     
     /**
+     * Test getControlBlocksForStage method.
+     */
+    @Test
+    public void testGetControlBlocksForStage()
+    {
+        setUpSubmission(submissions[1]);
+        submissions[1].countBlockCategoriesForStage();
+
+        int expected = 1;
+        int actual = submissions[1].getControlBlocksForStage();
+        assertEquals("should be equal", expected, actual);
```

Comment:
Could this be reduced to one line i.e.:
assertEquals("should be equal", 1, submissions[1].getControlBlocksForStage()); 
?